spec/controllers/security_group_controller_spec.rb

```diff
@@ -67,9 +66,7 @@
       allow_any_instance_of(SecurityGroup).to receive(:delete_security_group_queue)
     end
 
-    subject do
-      get :delete_security_groups, :params => {:id => security_group.id}
-    end
+    subject { get :delete_security_groups, :params => {:id => security_group.id} }
```

Comment:
Can you please use parentheses around the `get` method? It is a little confusing with the one-lined block.